cuda/runtime/BUILD.local_cuda

```diff
@@ -52,14 +55,15 @@ cc_library(
 cc_import(
     name = "cudart_lib",
     interface_library = "cuda/lib/x64/cudart.lib",
-    system_provided = 1,
+    shared_library = "cuda/bin/cudart64_"+ str(cuda_package_version("cuda_cudart")) + ".dll",
```

Comment:
This seems like it might be x64 specific from the name? Do you need to also provide the 32 bit variants of these libraries or are they always only 64 bit? I'm not so familiar with cuda libraries on windows.
If its x64 only then the `target_compatible_with` option will also need updating.